bokeh/_templates/file.html

```diff
@@ -3,11 +3,12 @@
     <head>
         <meta charset="utf-8">
         <title>{{ title if title else "Bokeh Plot" }}</title>
-        {{ plot_resources|indent(8) }}
-
-        {{ plot_script|indent(8) }}
+        {{ bokeh_css }}
     </head>
     <body>
         {{ plot_div|indent(8) }}
+
+        {{ bokeh_js }}
```

Comment:
I thought there was some reason we put bokehjs in `head` but I can't recall for sure